roles/network_plugin/weave/templates/weave-net.yml.j2

```diff
@@ -260,6 +266,9 @@ items:
             - name: cni-conf
               hostPath:
                 path: /etc
+            - name: cni-machine-id
+              hostPath:
+                path: /etc/machine-id
```

Comment:
Why doesn't here specify `type: FileOrCreate` which is specified on the original one?